pkg/artifactory/resource_artifactory_virtual_conan_repository.go

```diff
@@ -0,0 +1,46 @@
+package artifactory
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+type ConanVirtualRepositoryParams struct {
+	VirtualRepositoryBaseParams
+	VirtualRetrievalCachePeriodSecs int `hcl:"virtual_retrieval_cache_period_seconds" json:"virtualRetrievalCachePeriodSecs,omitempty"`
+}
+
+func resourceArtifactoryConanVirtualRepository() *schema.Resource {
+	var conanVirtualSchema = mergeSchema(baseVirtualRepoSchema, map[string]*schema.Schema{
+		"virtual_retrieval_cache_period_seconds": {
```

Comment:
Do we need the prefix `virtual_` here? The resource already contains the word `virtual`

Reply:
made it retrieval_cache_period_seconds now. ie: removed virtual in name as suggested.